test/common/ops/buy.js

```diff
@@ -59,7 +68,11 @@ describe('user.fns.buy', () => {
     it('does not purchase if not enough gp', () => {
       user.stats.hp = 45;
       user.stats.gp = 5;
-      user.ops.buy({params: {key: 'potion'}});
+      try {
```

Comment:
Pretty sure all these assertions don't actually work as you expect them to. 

The expect...throw syntax supposed to be written like:

```js
// function without arguments
let aFunctionWithoutArgsThatThrowsAnError = () => {/* do stuff, throw an error */};
let error = new NotAuthorized(); // might need message as well, not sure
expect(aFunctionWithoutArgsThatThrowsAnError).to.throw(error);

// function with arguments
let aFunctionWithArgsThatThrowsAnError = (arg1, arg2) => {/* do stuff, throw an error */};
let error = new NotAuthorized(); // might need message as well, not sure
expect(() => {
  aFunctionWithArgsThatThrowsAnError('foo', 'bar');
}).to.throw(error);
```

And it stops the error from being thrown. Meaning you don't need the try catch.

The way you have it written, I'm pretty sure the `expect...throw` assertion is never being evaluated. It's just throwing normally, hence why you needed the `try...catch`